### PR TITLE
Add timetable for weekend semester and etc.

### PR DESCRIPTION
**Pull request by Taewan-P**
- Add timetable for weekend semester
- Fixed column name for Yesulin Apt. timetable

### CODE REVIEW

File: timetable/semester/week/YesulIn_week.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "terminal_week": [
+  "yesulin_weekend": [
```
Comment (CXZ7720) [inline timetable/semester/week/YesulIn_week.json]: change weekend to week plz.

Comment (Taewan-P) [reply timetable/semester/week/YesulIn_week.json]: Fast

Comment (CXZ7720) [reply timetable/semester/week/YesulIn_week.json]: thanks. :)

**Comment by CXZ7720** [comment]
Why don't you change https://github.com/CXZ7720/shuttlecock_API/blob/f173c3a6fe349db6e049528c8aa7f4ad0c25733b/router/yesulin.js#L38
same as the column name that you updated?

**Pull request merged**